### PR TITLE
Added GA promos

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**
The copy for this promo comes from - https://github.com/neo4j-graphacademy/courses/blob/main/asciidoc/courses/importing-fundamentals/promo.adoc

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-data-importer-29.surge.sh

Updated pages:
- [Other ways of importing data into Neo4j](https://neo4j-docs-data-importer-29.surge.sh/data-importer/1.0/import-others)
- [Neo4j Data Importer](https://neo4j-docs-data-importer-29.surge.sh/data-importer/1.0)

File: modules/ROOT/pages/import-others.adoc [hunk 1]
```diff
@@ -24,4 +24,7 @@ See link:https://neo4j.com/docs/getting-started/languages-guides/integration-too
 
 If your data is not in a CSV format, using APOC Load procedures allows you to ingest data from formats including json, xml and arrow.
 APOC is a library of user-defined procedures and functions that extends the use of Cypher.
-See link:https://neo4j.com/docs/apoc/current/overview/apoc.load/[APOC user guide -> apoc.load] for more information about the load procedure.
\ No newline at end of file
+See link:https://neo4j.com/docs/apoc/current/overview/apoc.load/[APOC user guide -> apoc.load] for more information about the load procedure.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/import-others.adoc:27]: Can it say _Explore data importing options with the [Importing Data Fundamentals course on GraphAcademy](https://graphacademy.neo4j.com/courses/importing-fundamentals/?ref=docs-promo-import)._ instead? (Ie drop the _your_)

**Review by AlexicaWright — COMMENTED** [review]
A suggestion.

File: modules/ROOT/pages/import-others.adoc (hunk 1 above)
Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/import-others.adoc:27]: https://github.com/neo4j-graphacademy/courses/pull/226

**Review by AlexicaWright — APPROVED** [review]

**Pull request merged**